lib/cassandra.rb

```diff
@@ -278,6 +356,12 @@ def self.cluster_async(options = {})
         raise ::ArgumentError, "both :client_cert and :private_key options must be specified, but only :private_key given"
       end
 
+      Util.assert_instance_of(::String, options[:client_cert]) {
+        ":client_cert must be a string, #{options[:client_cert].inspect} given"
+      }
+      Util.assert_instance_of(::String, options[:private_key]) {
+        ":client_cert must be a string, #{options[:private_key].inspect} given"
+      }
```

Comment:
I added this condition so that we get a better error when the option value is nil. I could allow for String or Pathname, but are there other classes I'd want to white list? Or should I black-list nil instead of white-listing?